src/Symfony/Component/HttpFoundation/Tests/RateLimiter/AbstractRequestRateLimiterTest.php

```diff
@@ -33,6 +33,26 @@ public function testConsume(array $rateLimits, ?RateLimit $expected)
         $this->assertSame($expected, $rateLimiter->consume(new Request()));
     }
 
+    public function testConsumeWithoutLimiterAddSpecialNoLimiter()
```

Comment:
This test fails on the low-deps run: https://github.com/symfony/symfony/actions/runs/9989450887/job/27608100847#step:7:1761

Reply:
This is an underlying bug in 5.2.0, where `\INF` was used before being changed to `\PHP_INT_MAX`: https://github.com/symfony/symfony/blob/v5.2.0/src/Symfony/Component/RateLimiter/Policy/NoLimiter.php#L37. I skip the test when NoLimiter is buggy. What do you think?